timesketch/lib/experimental/create_graph.cql

```diff
@@ -76,7 +76,7 @@ WITH line
 
   MERGE (src:WindowsMachine { hostname: line.src, sketch_id: {sketch_id} })
   MERGE (service:WindowsService { service_name: line.svc_name, sketch_id: {sketch_id} })
-  MERGE (image_path:WindowsServiceImagePath { image_path: line.image_path, sketch_id: {sketch_id} })
+  MERGE (image_path:WindowsServiceImagePath { image_path: line.image_path, image_path_short: line.image_path_short, sketch_id: {sketch_id} })
```

Comment:
Do we need this in the MERGE, wouldn't line.image_path be enough end then just set image_path_short in the SET?

Reply:
Ok. I will fix it in one of next commits.

Reply:
ack

Reply:
Actually, putting it in SET is avoids one bug.